libs/datamodel/connectors/sql-datamodel-connector/src/postgres_datamodel_connector.rs

```diff
@@ -140,6 +141,24 @@ impl Connector for PostgresDatamodelConnector {
                     _ => {}
                 }
             }
+            if native_type_name == SMALL_SERIAL_TYPE_NAME
```

Comment:
I've seen some larger ifs in your PRs, if you like, you can do `matches!` as well to not have to do `native_type_name ==` all the time:
```rust
if matches!(native_type_name, SMALL_SERIAL_TYPE_NAME | SERIAL_TYPE_NAME | BIG_SERIAL_TYPE_NAME) {
 // ...
}
```
Not really a requirement, more a flavor question.

Reply:
Oh that's neat! Didn't know about that.